### ISSUE TITLE
Pixi 4 Error

### ISSUE THREAD

**Issue by dan335** [gluckgames/pixi-packer-parser#3]
I tried using pixi.js v4.0.0-rc2 but got this error.

```
Uncaught Error: Texture Error: frame does not fit inside the base Texture dimensions [object Object]
```

3.0.11 works without error.

**Comment by marekventur** [comment]
There has been a lot of back and forth around textures in the PIXI dev branch (it's far from stable in that area), that's why I haven't made any changes yet. I'm happy to accept PRs if you can find a working solution.

Did you check pixi out the v4.0.0-rc2 tag or the head of the dev branch? I was hoping https://github.com/pixijs/pixi.js/commit/17358e1bc3d32b9d5791ba98854b66f5c5300ef1 would solve the problem, but I haven't tried it yet.

**Comment by dan335** [comment]
It was rc2 I tried before.  Just tried pixi.min.js from the dev branch and got same error.

**Comment by marekventur** [comment]
Ok. We're waiting for v4.0 final before switching, so it'll be probably a bit before I'll find the time to fix this properly. If you find a solution before me, please share!

**Comment by dan335** [comment]
Got it working with 4.0. https://github.com/Gamevy/pixi-packer-parser/pull/4

**Comment by dan335** [comment]
That PR also fixed the error I had.

**Comment by marekventur** [comment]
released as v1.2.0
